crowbar_framework/lib/crowbar/chef/upload.rb

```diff
@@ -69,6 +69,15 @@ def data_bags
           loader = ::Chef::Knife::Core::ObjectLoader.new(::Chef::DataBagItem, logger)
           data_bags = loader.find_all_object_dirs(chef_data_bags_path) || []
           data_bags.each do |data_bag|
+            begin
+              api.post_rest("data", name: data_bag)
+            rescue Net::HTTPServerException => e
+              unless e.response.code == 409
```

Comment:
not sure but shouldn't it be an `if` instead of an `unless`? or why do you skip when you get 409?